config/pro_errors.yml

```diff
@@ -0,0 +1,18 @@
+en:
```

Comment:
Probably not for this PR, but I would consider moving both `config/errors.yml` and `config/pro_errors.yml` to the `config/locales` folder.